README.md

```diff
@@ -112,6 +112,15 @@ In Powershell:
 
 Note that OpenSSL isn't needed on Windows.
 
+### Setup on OpenBSD
```

Comment:
There isn't a really strict requirement from the base system, rather the ports/packages have to be up to date. And base/packages out of sync is no go. So `OpenBSD 6.0` would be more accurate.